skills/const-documenter/SKILL.md

```
@@ -0,0 +1,197 @@
+
+## レビュー後の作業
+
+```bash
+# 未レビューの定数を検索
+grep -r "@todo 要確認" src/Constants/
+
+# 確信度低のみ検索
+grep -r "確信度低" src/Constants/
+```
+
+レビュー完了後、`@todo 要確認(確信度X): ` 部分を削除してコメントのみ残す。
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)